code/modules/projectiles/guns/projectile/battle_rifle/type47.dm

```diff
@@ -17,14 +17,15 @@
 	recoil_buildup = 2
 	penetration_multiplier = 1.5
 	damage_multiplier = 1.5
-	one_hand_penalty = 10 
+	one_hand_penalty = 10
 	gun_tags = list(GUN_SILENCABLE)
 	gun_parts = list(/obj/item/part/gun = 2 ,/obj/item/stack/material/plasteel = 6)
 	init_firemodes = list(
 		SEMI_AUTO_NODELAY,
 		BURST_3_ROUND
 		)
-	spawn_blacklisted = TRUE //until loot rework
```

Comment:
Sneaky unblacklist of the type47?

Reply:
i love fixing merge conflicts i love fixing merge conflicts i love fixing merge conflicts